src/controllers/v1/communication.js

```diff
@@ -0,0 +1,45 @@
+const communicationService = require('@services/communication')
+
+module.exports = class Communication {
+	async signup(req) {
+		try {
+			const sendEmail = await communicationService.signup(req.body)
+			return sendEmail
+		} catch (error) {
+			return error
+		}
+	}
+
+	async login(req) {
+		try {
+			const sendEmail = await communicationService.login(req.body)
```

Comment:
@nevil-mathew add api validation for the required fields

Reply:
Validators are pending for all new apis, can we take all this in a single shot please.